lib/provider/tests/test_boku.py

```diff
@@ -38,7 +38,7 @@ def test_good(self):
     @patch('lib.boku.client.mocks', {'verify-trx-id': (500, '')})
     def test_verify_fails(self):
         self.add_seller_boku()
-        eq_(self.client.post(self.url, data=self.sample()).status_code, 200)
+        self.client.post(self.url, data=self.sample()).status_code
```

Comment:
Why do you need to access ``.status_code`` ? Does that force-parse the response or something? If so, is there a more explicit way to do that?